openlibrary/plugins/upstream/models.py

```diff
@@ -832,6 +832,10 @@ def update_loan_status(self):
         for loan in loans:
             lending.sync_loan(loan['ocaid'])
 
+    def get_safe_mode(self):
+        settings = self.get_users_settings()
+        return settings['safe_mode'] if 'safe_mode' in settings.keys() else ''
```

Comment:
https://docs.python.org/3/library/stdtypes.html#dict.get
```suggestion
        return self.get_users_settings().get('safe_mode', '')
```